src/main/java/com/linkedin/kmf/partitioner/OldByteArrayKMPartitioner.java

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright 2016 LinkedIn Corp. Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.linkedin.kmf.partitioner;
+
+import java.util.Arrays;
+
+public class OldByteArrayKMPartitioner implements KMPartitioner {
+  public int getPartitionForKey(String key, int partitionNum) {
```

Comment:
I think we don't need it. We have added `OldKMPartitioner` and `NewKMPartitioner` because we have use-cases for both. It should be safe to remove this `OldByteArrayKMPartitioner` and only add it if it is useful.